packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -440,3 +440,5 @@ export class Wizard extends React.Component<WizardProps> {
     );
   }
 }
+
+export default Wizard;
```

Comment:
Why do we need these `export default`s for Wizard?

Reply:
It's mainly if they wanted to do import Wizard from './Wizard'.  However since this isn't how we recommend importing these I'll remove this.  Thanks.